docs/components/card.md

```diff
@@ -1,275 +1,179 @@
-
-<KCard :hasHover="true">
-  <template v-slot:body>
-    <KAlert alert-message="Welcome to Kong!" />
-    <div>
-      <div style="display: flex; justify-content: space-between; align-items: center;">
-        <h2>Kong Enterprise Edition</h2>
-        <KButton appearance="secondary" to="https://docs.konghq.com/enterprise" target="_blank">
-          Docs
-        </KButton>
-      </div>
-      <div>
-        <p>Kong Enterprise adds features, functionality, and performance to Kong. This documentation doesn’t cover the general practices that are common to both Kong and Kong Enterprise—learn the basics in Kong documentation.</p>
-      </div>
-    </div>
+  <template #actions>
+    <KDropdown
+      :items="[
+        { label: 'Props', to: { path: '/components/card#props' } },
+        { label: 'Slots', to: { path: '/components/card#slots' } }
+      ]"
+    >
+      <KButton
+        appearance="tertiary"
+        icon
+        size="small"
+      >
+        <MoreIcon />
+      </KButton>
+    </KDropdown>
   </template>
-</KCard>
-
-```html
-<KCard :hasHover="true">
-  <template v-slot:body>
-    <KAlert alert-message="Welcome to Kong!" />
-    <div>
-      <div>
-        <h2>Kong Enterprise Edition</h2>
-        <KButton appearance="secondary" to="https://docs.konghq.com/enterprise" target="_blank">
-          Docs
-        </KButton>
-      </div>
-      <div>
-        <p>Kong Enterprise adds features, functionality, and performance to Kong. This documentation doesn’t cover the general practices that are common to both Kong and Kong Enterprise—learn the basics in Kong documentation.</p>
-      </div>
+  Lorem ipsum dolor sit amet ...
+  <template #footer>
+    <div class="horizontal-spacing">
+      <KBadge appearance="success">
+        Published
+      </KBadge>
+      <KBadge>2 versions</KBadge>
     </div>
   </template>
 </KCard>
 ```
 
-### status
-
-String to be used in the `statusHat` slot.
-
-<KCard status="My status" title="My title" body="My body" />
-
-```html
-<KCard status="My status" title="My title" body="My body" />
-```
-
-### body
-
-String to be used in the body slot.
-
-<KCard body="I am the body."/>
-
-```html
-<KCard body="I am the body."/>
-```
-
-### borderVariant
-
-Sets top border or no border. If neither set default will have border
-
-- `borderTop`
-- `noBorder`
-
-<div class="borderless-cards">
-  <KCard title="Card without border" body="Body Content" border-variant="noBorder"/>
-
-  <KCard title="Card with top border" body="Body Content" border-variant="borderTop"/>
-</div>
-
-```html
-<KCard title="Card without border" body="Body Content" border-variant="noBorder"/>
-
-<KCard title="Card with top border" body="Body Content" border-variant="borderTop"/>
-```
+## Props
 
-### hasHover
+### title
 
-Set if card should have shadow state (shadow) on hover
+String to be used as a title. Can also be [slotted](#slots).
 
-<KCard title="hasHover" body="This card only has a shadow on hover" has-hover />
+### content
```

Comment:
question: does it make sense to keep the `content` prop instead of just requiring the user to slot the card content?

Wondering if we ever use this